wdl2cwl/tests/wdl_files/hisat2.wdl

```diff
@@ -1,4 +1,5 @@
 version 1.0
+# Source: https://github.com/biowdl/tasks/blob/develop/hisat2.wdl
```

Comment:
It is a bit safer to use the "permalink" version. On almost any GitHub page, if you press the `y` key on your keyboard the URL will change to a permanent version. In this case it is https://github.com/biowdl/tasks/blob/bc1bacf11498d2d30b85591cfccdcf71ef0966a5/hisat2.wdl